### DISCUSSION TITLE
需求case

### DISCUSSION THREAD

**Discussion by icbd** [geeknote-net/feedback#10] [Ideas]
目前我使用 Evernote 来管理笔记, 用 GitHub Page 来管理博客(Jekyll).

笔记和博客在需求上有一些共同点, 比如流畅的编写体验, 便捷的分享功能, 全文搜索, 导出备份等.
笔记期待特别的安全策略, 比如文本加密传输和保存.
博客需要一点社交互动, 以及 SEO 优化.

以上是我个人的需求.

祝 GeekNote 越做越好!

**Comment by chloerei** [comment]
虽然名字带有 Note，但本质是个 Blog。xxblog 的域名都抢不到了，所以选了 GeekNote 这个比较好记的名字。

因为博客内容主要是为了公开的， 所以加密存储这类功能应该不会做，不然搜索功能就做不了，也不建议在上面存放需要保密的信息。

导出功能会做。

感谢反馈🙏

**Reply by duron600** [reply]
是否考虑允许用户不公开自己的文章呢？

**Reply by chloerei** [reply]
有考虑过像 YouTube 那样支持“不公开列出”和“私享”，但没决定，不知道这个有多少需求。

可以描述一下什么场景需要这样的功能吗？

**Reply by duron600** [reply]
不是什么特别的场景，就是自己有时候写的东西算是一些生活上的总结和反思，不一定是打算发出来跟大家分享的。有些性格内向的人可能比较倾向于这么做。也可能有时候写得太过理性，在别人看来可能过于极端，会让别人不舒服，也容易引起争吵，所以不希望让人看到。

有可能纯粹是我个人的需求，如果本来没太大打算的话也不用太在意。

**Reply by chloerei** [reply]
还想到一种实现，添加一个“归档“状态，就是写完了不发布也不删除，移到归档不出现在草稿箱。

”不公开列出”，“私享”（白名单访问）和“归档”（不发布）哪个更贴近你的需求呢？

**Reply by duron600** [reply]
对我来说前二者都可以吧，似乎“私享”从功能上覆盖了“不公开列出”，但只有“不公开列出”也足够。

**Reply by chloerei** [reply]
好的，这个需求先记录一下，之后一并考虑。
